tests/chain_integration/test_currency_network.py

```diff
@@ -131,9 +127,6 @@ def test_get_transfer_event(currency_network_with_events, accounts, test_extra_d
     assert transfer_event.value == 10
     assert transfer_event.to == accounts[0]
     assert transfer_event.from_ == accounts[1]
-    assert transfer_event.user == accounts[0]
-    assert transfer_event.counter_party == accounts[1]
-    assert transfer_event.direction == "received"
```

Comment:
Why was this removed?

Reply:
Sry I trimmed it to the contract event parameters. But they haven't been touched and those are artificial parameter by the relay as I understand now. I recovered them.